src/i18n.js

```diff
@@ -4,33 +4,36 @@
  * License: MIT
  */
 
-import i18next from "i18next";
-
-i18next.init({
-  interpolation: {
-    escapeValue: false
+const i18nConfig = {
+  "en-US": {
+    "Type the link and press enter": "Type the link and press enter",
+    "Invalid Link": "Invalid Link",
+    "Can't show plugin, component {{type}} not found.":
+      "Can't show plugin, component {{type}} not found.",
+    "Block List": "Block List"
   },
-  lng: "en-US", // "en-US" | "pt-BR"
-  resources: {
-    "en-US": {
-      translations: {
-        "Type the link and press enter": "Type the link and press enter",
-        "Invalid Link": "Invalid Link",
-        'Can\'t show plugin, component "{{type}}" not found.':
-          'Can\'t show plugin, component "{{type}}" not found.',
-        "Block List": "Block List"
-      }
-    },
-    "pt-BR": {
-      translations: {
-        "Type the link and press enter": "Digite o link e pressione enter",
-        "Invalid Link": "Link inválido",
-        'Can\'t show plugin, component "{{type}}" not found.':
-          'Não é possível exibir o plugin, componente "{{type}}" não encontrado',
-        "Block List": "Lista de Blocos"
-      }
-    }
+  "pt-BR": {
+    "Type the link and press enter": "Digite o link e pressione enter",
+    "Invalid Link": "Link inválido",
+    "Can't show plugin, component {{type}} not found.":
+      "Não é possível exibir o plugin, componente {{type}} não encontrado",
+    "Block List": "Lista de Blocos"
   }
-});
+};
+
+// export const t = obj => {
+//   const message =
+//     (obj.i18n && obj.i18n[obj.language] && obj.i18n[obj.language][obj.key]) ||
+//     i18nConfig[obj.language][obj.key];
+
+//   return message;
+// };
+
+export const replaceData = (str, data) => {
+  const rgx = /{{([A-z-$]+[\w]*)}}/gm;
```

Comment:
Simplifying this regex

https://regex101.com/r/fR1RWW/1
https://regex101.com/r/fR1RWW/2

or improving... 

https://regex101.com/r/fR1RWW/3